UI/src/app/services/upload.service.ts

```diff
@@ -28,24 +29,40 @@ export class UploadService {
 
   onFileChange(event: any): void {
     const files = event.target.files;
-    const dotPosition = files[0].name.lastIndexOf('.');
+    const dotPosition = files[ 0 ].name.lastIndexOf('.');
     this.uploadSchema(files).subscribe(res => {
       this.snackbar.open(
         'Success file upload',
         ' DISMISS ',
-        {duration: 3000}
+        { duration: 3000 }
```

Comment:
please remove duration, now it's 3000 by default

Reply:
Done